server/tslint.json

```diff
@@ -8,6 +8,7 @@
     "ordered-imports": [true, { "named-imports-order": "case-insensitive" }],
     "member-ordering": [true, { "order": [ "constructor" ] }],
     "interface-name": false,
-    "no-string-literal": false
+    "no-string-literal": false,
+    "prefer-object-spread": false
```

Comment:
why you add this?

Reply:
If prefer-object-spread is true I cannot use Object.assign in the way I did to merge object. This would lead to an lint error

Reply:
you could use the spread operator 
```js
{...recipe,...requestBody}
```
instead of 
```js
Object.assign(recipe, requestBody);
```

Reply:
Yes I tried it but it didn't work. I always need to something like

const variable = {...recipe,...requestBody}
const realData = variable._doc

Else I would get a object with a lot of meta data like cache things and so on.

Do you know why I do not get a simple Object after using spread?